### ISSUE TITLE
 "R_stringdist" not resolved from current namespace (stringdist)

### ISSUE THREAD

**Issue by ejyang06** [markvanderloo/stringdist#72]
I got an error as
```
Error in .Call("R_stringdist", a, b, method, as.double(weight), as.double(p),  : 
  "R_stringdist" not resolved from current namespace (stringdist)
```
How should I resolve this?
Thanks

**Comment by markvanderloo** [comment]
Hi,

You closed this call but it would be nice to know how you solved it, as it pops up for more users (see #73).

Could you elaborate what went wrong and how you solved it?

Thanks in advance!
Mark

**Comment by markvanderloo** [comment]
Closing this, see #73 and https://github.com/markvanderloo/stringdist/commit/e648fd8168234064bfe37ee180a19f7fe8abd1a6
